packages/components/src/Select/ListBox.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useRef } from 'react';
+import { useListBox } from 'react-aria';
+import type { AriaListBoxOptions } from '@react-aria/listbox';
+import type { ListState } from 'react-stately';
+
+import { Box } from '../Box';
+import { Option } from './Option';
+import { useStyles } from '@marigold/system';
+
+interface ListBoxProps extends AriaListBoxOptions<unknown> {
+  listBoxRef?: React.RefObject<HTMLUListElement>;
+  state: ListState<unknown>;
+}
+
+export const ListBox = (props: ListBoxProps) => {
+  const ref = useRef<HTMLUListElement>(null);
+  const { listBoxRef = ref, state } = props;
```

Comment:
Can you explain what is happening here? Why do you override it?

Reply:
I can make it better. Little bit to much copy/paste :)